extension.css

```diff
@@ -279,107 +272,102 @@ button.browser-style.expander {
-  border-color: #005bab;
-}
-
-button.browser-style.default:hover:active:not(.hover):not(.disabled):not(.focused) {
-  background-color: #005bab;
-  border-color: #004480;
-}
-
-button.browser-style:focus:not(.disabled) {
-  border-color: #fff !important;
-  box-shadow: 0 0 0 2px rgba(97, 181, 255, 0.75);
-}
-
 /* Fields */
 .browser-style > input[type="text"],
 textarea.browser-style {
   background-color: #fff;
-  border: 1px solid #b1b1b1;
-  box-shadow: 0 0 0 0 rgba(97, 181, 255, 0);
-  font: caption;
+  border: 1px solid var(--grey-20);
+  border-radius: 2px;
+  font: inherit;
+  margin-bottom: 6px;
   padding: 0 6px 0;
   transition-duration: 250ms;
   transition-property: box-shadow;
+  transition-timing-function: cubic-bezier(.07,.95,0,1);
 }
 
-.browser-style > input[type="text"] {
-  height: 24px;
-}
-
-.browser-style > input[type="text"].hover,
-textarea.browser-style.hover {
-  border: 1px solid #858585;
+textarea.browser-style {
+  padding: 6px;
 }
 
-.browser-style > input[type="text"].disabled,
-textarea.browser-style.disabled {
-  color: #999;
-  opacity: .5;
+.browser-style > input[type="text"] {
+  height: 32px;
 }
 
-.browser-style > input[type="text"].focused,
-textarea.browser-style.focused {
-  border-color: #0996f8;
-  box-shadow: 0 0 0 2px rgba(97, 181, 255, 0.75);
+.browser-style > input[type="text"]:not(:disabled):hover,
+textarea.browser-style:not(:disabled):hover {
+  border-color: var(--grey-30);
 }
 
-/* Interactive States */
-.browser-style > input[type="text"]:not(disabled):hover,
-textarea.browser-style:not(disabled):hover {
-  border: 1px solid #858585;
+.browser-style > input[type="text"]:disabled,
+textarea.browser-style:disabled {
+  opacity: .4;
 }
 
-.browser-style > input[type="text"]:focus,
-.browser-style > input[type="text"]:focus:hover,
-textarea.browser-style:focus,
-textarea.browser-style:focus:hover {
-  border-color: #0996f8;
-  box-shadow: 0 0 0 2px rgba(97, 181, 255, 0.75);
+.browser-style > input[type="text"]:not(:disabled):focus,
+textarea.browser-style:not(:disabled):focus,
+.browser-style > input[type="text"]:not(:disabled):active,
+textarea.browser-style:not(:disabled):active {
+  border-color: var(--blue-50);
+  box-shadow: 0 0 0 1px var(--blue-50), 0 0 0 4px var(--blue-50-a30);
 }
 
 /* stylelint-disable property-no-vendor-prefix */
 .panel-section {
   display: flex;
   flex-direction: row;
+  padding: 16px;
 }
 
 .panel-section-separator {
-  background-color: rgba(0, 0, 0, 0.15);
+  background-color: var(--grey-30);
   min-height: 1px;
 }
 
 /* Panel Section - Header */
 .panel-section-header {
-  border-bottom: 1px solid rgba(0, 0, 0, 0.15);
-  padding: 16px;
+  align-items: center;
+  border-bottom: 1px solid var(--grey-30);
+  display: flex;
+  flex: 1 auto;
+  height: 40px;
+  padding: 4px;
 }
 
 .panel-section-header > .icon-section-header {
+  -moz-context-properties: fill;
+  background-color: transparent;
+  background-image: url(chrome://browser/skin/arrow-left.svg);
```

Comment:
I'm not sure we can load `chrome://` images from content pages…

Reply:
It actually works 😅 I now asked the webextension team is it's intended. My rational was to avoid to load the same icon once again. Alternatively icons tokens wanted! 😉